### PR TITLE
fix convertor

### PR DESCRIPTION
**Pull request by fabienjuif**

### CODE REVIEW

**Comment by fabienjuif** [comment]
Put back the rotation rendering : `graphics.rotation = body.angle` (client.js)
You'll see a little missmatch.

This is how I saw I had some problems at first.

File: src/physic/engine.js [hunk 1]
```diff
@@ -15,21 +15,32 @@ export const addBodies = (...bodies) => {
...
+  world.solver.tolerance = 0.001
+  world.setGlobalStiffness(1e4)
+
   addBodies(...createWalls(WIDTH, HEIGHT))
 
   return world
 }
 
 let lastTime
+const maxSubSteps = 2 // Max physics ticks per render frame
```
Comment (fabienjuif) [inline src/physic/engine.js:33]: Why lowering down this value ?

Comment (bpetetot) [reply src/physic/engine.js:33]: it was only for testing purpose

File: src/renderer/entities/box.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ export const renderBox = (body) => {
   const graphics = new Graphics()
   graphics.beginFill(randomColor())
   graphics.drawRect(x, y, w, h)
+  graphics.pivot.set(w / 2, h / 2)
```
Comment (fabienjuif) [inline src/renderer/entities/box.js:11]: Yeah nice, I prefers it this way :)

**Pull request merged**